documentation/puppet-api/v3/all_environment_modules.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema":     "http://json-schema.org/draft-04/schema#",
+  "title":       "All Environment Modules",
+  "description": "An array of information about all the modules in all Puppet code environments",
+  "type":        "array",
+  "properties": {
+    "$ref": "#/environment_modules"
+  },
+  "additionalProperties": false
+}
```

Comment:
I think that comment got deleted when I rebased...but I think this passes the validator according to http://www.jsonschemavalidator.net/ using the draft 04 validation.